ee/clickhouse/test/test_process_event_ee.py

```diff
@@ -266,6 +266,26 @@ def test_ip_capture(self) -> None:
         events = get_events()
         self.assertEqual(events[0]["properties"]["$ip"], '"11.12.13.14"')
 
+    def test_ip_override(self) -> None:
+        user = self._create_user("tim")
+        Person.objects.create(team=self.team, distinct_ids=["asdfasdfasdf"])
+
+        process_event_ee(
+            "asdfasdfasdf",
```

Comment:
Definitely a nitpick (and not specifically related to this PR), but these "asdfasdfasdf" event names, IDs, etc seem to be all over the tests. Probably helpful for newcomers + nicer aesthetically to have them actually be a little more descriptive. You can see it generates confusion because the test `test_capture_no_sent_at` actually passes "movie played" as the `distinct_id`